auth/api/api.go

```diff
@@ -95,7 +125,86 @@ func OAuthCallback(w http.ResponseWriter, r *http.Request) {
 	render.JSON(w, r, userData)
 }
 
+func InitGrantOAuth(w http.ResponseWriter, r *http.Request) {
+	ctx, cancel := context.WithCancel(r.Context())
+	defer cancel()
+
+	grantType := chi.URLParam(r, "grant")
+	var user GrantOAuthUserRequestBody
+	if parseBodyErr := json.NewDecoder(r.Body).Decode(&user); parseBodyErr != nil {
+		log.Error().Err(parseBodyErr).Msg("failed to parse the request payload")
+		_ = render.Render(w, r, apiutils.BadRequest("invalid payload"))
+		return
+	}
+
+	hashedPassword, hashingErr := hashing.Hash(user.Password)
+	if hashingErr != nil {
+		log.Error().Err(hashingErr).Msg("failed to encrypt the password longer than 72 bytes")
+		_ = render.Render(w, r, apiutils.BadRequest("invalid password size"))
+		return
+	}
+
+	log.Info().Msgf("hashed passcode: %s", hashedPassword)
```

Comment:
we defintely shouldnt be logging this stuff - maybe on debug, but not info. I would also not do this on debug frankly.

Reply:
my bad. I added this stuff for myself, I'll remove these.